app/furniture/journal/keyword.rb

```diff
@@ -6,10 +6,16 @@ class Keyword < ApplicationRecord
     validates :canonical_keyword, presence: true, uniqueness: {case_sensitive: false, scope: :journal_id}
     belongs_to :journal, inverse_of: :keywords
 
+    scope(:search, lambda do |*keywords|
+      where("lower(aliases::text)::text[] && ARRAY[?]::text[]", keywords.map(&:downcase))
+          .or(where("lower(canonical_keyword) IN (?)", keywords.map(&:downcase)))
```

Comment:
There is probably a way less awkward way of doing this; but this is what I was able to come up with 🙃